Bottom: 1deb57886fd08c0af3c3b23e148be9a4088b385a
Top:    ab8c9f6e43cec4fe6a2baf10b09b659b57153b04
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-27 04:50:05 +0100

Refresh of crasy-test

---

diff --git a/src/draw_funcs.c b/src/draw_funcs.c
index 4bed53b..f63c0f4 100644
--- a/src/draw_funcs.c
+++ b/src/draw_funcs.c
@@ -251,10 +251,10 @@ pad_inlayer_callback (const BoxType * b, void *cl)
 {
   PadType* pad = (PadType *)b;
   LayerType *layer = cl;
-  int solder_group = GetLayerGroupNumberByNumber (solder_silk_layer);
+  int bottom_group = GetLayerGroupNumberBySide (BOTTOM_SIDE);
   int group = GetLayerGroupNumberByPointer (layer);
 
-  int side = (group == solder_group) ? SOLDER_LAYER : COMPONENT_LAYER;
+  int side = (group == bottom_group) ? BOTTOM_SIDE : TOP_SIDE;
 
   if (ON_SIDE (pad, side))
     {
@@ -344,8 +344,8 @@ pad_callback (const BoxType * b, void *cl)
 static void
 draw_ppv (int group, const BoxType *drawn_area, void *userdata)
 {
-  int component_group = GetLayerGroupNumberByNumber (component_silk_layer);
-  int solder_group = GetLayerGroupNumberByNumber (solder_silk_layer);
+  int top_group = GetLayerGroupNumberBySide (TOP_SIDE);
+  int bottom_group = GetLayerGroupNumberBySide (BOTTOM_SIDE);
   int side;
 
   if (PCB->PinOn || !gui->gui)
@@ -354,15 +354,15 @@ draw_ppv (int group, const BoxType *drawn_area, void *userdata)
       r_search (PCB->Data->pin_tree, drawn_area, NULL, pin_callback, NULL);
 
       /* draw element pads */
-      if (group == component_group)
+      if (group == top_group)
         {
-          side = COMPONENT_LAYER;
+          side = TOP_SIDE;
           r_search (PCB->Data->pad_tree, drawn_area, NULL, pad_callback, &side);
         }
 
-      if (group == solder_group)
+      if (group == bottom_group)
         {
-          side = SOLDER_LAYER;
+          side = BOTTOM_SIDE;
           r_search (PCB->Data->pad_tree, drawn_area, NULL, pad_callback, &side);
         }
     }
@@ -386,8 +386,8 @@ draw_holes (int plated, const BoxType *drawn_area, void *userdata)
 static void
 draw_layer (LayerType *layer, const BoxType *drawn_area, void *userdata)
 {
-  int component_group = GetLayerGroupNumberByNumber (component_silk_layer);
-  int solder_group = GetLayerGroupNumberByNumber (solder_silk_layer);
+  int top_group = GetLayerGroupNumberBySide (TOP_SIDE);
+  int bottom_group = GetLayerGroupNumberBySide (BOTTOM_SIDE);
   int layer_num = GetLayerNumber (PCB->Data, layer);
   int group = GetLayerGroupNumberByPointer (layer);
   struct poly_info info = {drawn_area, layer};
@@ -425,10 +425,10 @@ draw_layer (LayerType *layer, const BoxType *drawn_area, void *userdata)
   r_search (PCB->Data->pin_tree, drawn_area, NULL, pin_inlayer_callback, layer);
 
   /* draw element pads */
-  if (group == component_group)
+  if (group == top_group)
     r_search (PCB->Data->pad_tree, drawn_area, NULL, pad_inlayer_callback, layer);
 
-  if (group == solder_group)
+  if (group == bottom_group)
     r_search (PCB->Data->pad_tree, drawn_area, NULL, pad_inlayer_callback, layer);
 
   /* draw vias */
